pkg/cvo/status.go

```diff
@@ -274,24 +274,24 @@ func (optr *Operator) syncStatus(original, config *configv1.ClusterVersion, stat
 // from the cache (instead of clearing the status).
 // if ierr is nil, return nil
 // if ierr is not nil, update OperatorStatus as Failing and return ierr
-func (optr *Operator) syncFailingStatus(config *configv1.ClusterVersion, ierr error) error {
+func (optr *Operator) syncFailingStatus(original *configv1.ClusterVersion, ierr error) error {
```

Comment:
this change just renaming the variable... ?

Reply:
Ah, I see https://github.com/openshift/cluster-version-operator/pull/146/commits/1da7d0754c6797878db0edde23ec0f24bb299951#diff-6aba50cdad3ec2c1202f7ba49ed0314bR294